.github/workflows/build.yml

```diff
@@ -1,6 +1,6 @@
 name: Build
 
-on: [push]
+on: [push, pull_request]
```

Comment:
Just noticed that @adispring has attempted to fix this in a different way. I went for a "lazy" approach and thought we could run the CI pipeline anytime a code change is suggested (regardless of the destination). If it's cheap and quick I can't see any an obvious reason to restrict this to the main branch only.

Any objections?